service/service.go

```diff
@@ -65,67 +58,29 @@ func GrpcConnection(client, server string, insecureAllowed bool) (*grpc.ClientCo
 			return nil, errors.Errorf("Unknown Server %s:", server)
 		}
 	case FlowClient:
-		isTLS = false
```

Comment:
I think we need to set `insecureAllowed = true` here

Reply:
or alternatively keep `isTLS` above, and set it to `false` in this case and `!insecureAllowed` in the `LocalClient` case; and pass `isTLS` to `dial` instead of `insecureAllowed` (and fix the logic in there so we use tls config when `isTLS` and `WithInsecure` otherwise)

(when we're running the lingo client on the flow server, we connect directly to `localhost:8002`, but there's no cert declaring it's really localhost, so we can't use TLS; but we don't care, because, hey, it's localhost)

Reply:
(I think I'd prefer the second approach, it's more explicit)